### ISSUE TITLE
[Suggestion] GatsbyJS integration?

### ISSUE THREAD

**Issue by veerjainATgmail** [Urigo/WhatsApp-Clone-Client-React#772]
There are not many tutorials on how to use Gatsby for any dynamic web applications. 
It would be good to extend the this example?

**Comment by Urigo** [comment]
@veerjainATgmail why did you close this?
I do think it would be great to demonstrate using GatsbyJS here

**Comment by veerjainATgmail** [comment]
@Urigo, yes.. as some of the potential optimizations are already handled in the current code base and not many Gatsby dynamic application open source examples, i wasnt sure how much the Gatsby approach will benefit this whatsapp clone app.
